### PR TITLE
Hugo blog

### PR DESCRIPTION
**Pull request by artur-rs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,3 +204,43 @@ version of it. Now, you can use it in your article.
 >In case you don't have access to uploading images in Wordpress, contact
 Wordpress Administrator.
 >Remember to ALWAYS optimize all your images! It impacts the score heavily.
+
+## Publishig post
```
Comment (pietrushnic) [inline README.md]: @artur-rs this procedure doesn't seem to use news-and-ideas repositor. We should have procedure starting with `git clone <this_repo>`

**Comment by pietrushnic** [comment]
@artur-rs 13k SLOC changed, I hope it will not blow out everything :)

**Comment by pietrushnic** [comment]
@artur-rs @macpijan @miczyg1 i restructured README.md please take a look and let me know if I messed up something.

**Comment by artur-rs** [comment]
@pietrushnic looks ok

**Comment by miczyg1** [comment]
@pietrushnic looks allright to me, approved

**Comment by pietrushnic** [comment]
@macpijan please take look and merge

**Pull request merged**